_posts/2017/2017-09-12-five-compelling-reasons-why-you-should-care-about-terminology.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "5 compelling reasons why you should manage your terminology"
```

Comment:
It is hard to identify all 5 reasons at first sight. The last three headings clearly point out the benefits, but the first two headings aren't that clear. You only get the "reason" by reading the paragraph.